brave/src/test/java/brave/secondary_sampling/integration/SecondarySamplingIntegratedTest.java

```diff
@@ -18,10 +18,8 @@
 import brave.propagation.Propagation;
 import brave.sampler.RateLimitingSampler;
 import brave.sampler.Sampler;
-import brave.secondary_sampling.FakeHttpRequest;
-import brave.secondary_sampling.SamplerController;
-import brave.secondary_sampling.SecondarySampling;
-import brave.secondary_sampling.TraceForwarder;
+import brave.secondary_sampling.*;
```

Comment:
only thing is we don't do wildcard imports. probably your IDE is setup to do that. there's a Square IDE formatter mentioned in .github/CONTRIBUTING.md